manifests/p/Pallhed/LiveresultatDesktop/1.1.0/Pallhed.LiveresultatDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Pallhed.LiveresultatDesktop
+PackageVersion: 1.1.0
+PackageLocale: en-US
+Publisher: Pallhed
+PackageName: LiveresultatDesktop
+License: Unlicense

```

Comment:
Is this meant to be `Unlicensed` ? 
If so, then I would suggest that it be labeled `Freeware` instead